bin/twinkle-parser.js

```diff
@@ -21,11 +21,17 @@ Options:
 -p / --pretty : Prettify output
 --fields : Fields to be included (comma-separated, specifing all if not set)
 -h / --help : See this help
+-v / --version : Print version info
 `
   )
   process.exit(0)
 }
 
+else if (argv.v || argv.version) {
+  consola.log('v' + process.env.npm_package_version)
```

Comment:
I think, in most cases it works fine, but when this package is used as a child package, it may be value of the parent package. I'm not sure but I think it may be better to use `require('../package.json').version`. Tell me if you have any opinions.

https://docs.npmjs.com/cli/v7/using-npm/scripts#packagejson-vars